mantidproject/mantid
# Add diagnostics tab to sans gui

Description of work.
The diagnostic tab is a feature of the old SANS GUI which needs to be ported across to the new one.
In the old GUI this used the SumRowColumns Algorithm which only worked for LOQ. It has now been updated to use the ConvertToRealSpaceAxes and SumSpectra algorithms on order to cope with the tubes used by other SANS instruments.

**To test:**

- Code Review

- Find the loqdemo folder in the example data from here http://download.mantidproject.org/

- Open ISIS SANS V2 GUI
- Load the user files from loqdemo

- Switch to the diagnostic tab

- Select LOQ74044 as the file
- Click on Horizontal integral and verify that the below graph is plotted.



Fixes #21404 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
